pkg/pgclient/client.go

```diff
@@ -3,6 +3,10 @@ package pgclient
 import (
 	"context"
 	"fmt"
+	"github.com/timescale/promscale/pkg/pgmodel/cache"
+	"github.com/timescale/promscale/pkg/pgmodel/ingestor"
+	querier2 "github.com/timescale/promscale/pkg/pgmodel/querier"
```

Comment:
Is there a good reason for `querier2` naming? I'd rather change the name of the local variable its clashing with instead of the import.